Bottom: 15581548a22bb5f14df85582ea81fce947ba4672
Top:    9fae240addec441ce62b3299fe3fbcf58f2e8929
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:26:12 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index b529712..da12b9a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -155,9 +155,11 @@ pline_dump (VNODE * v)
   do
     {
       n = NEXT_VERTEX(v);
-      pcb_fprintf (stderr, "Line [%#mS %#mS %#mS %#mS 10 10 \"%s\"]\n",
+      pcb_fprintf (stderr, "Line [%$#mn %$#mn %$#mn %$#mn 10 10 \"%s\"] # %s, radius %mn\n",
 	       v->point[0], v->point[1],
-	       n->point[0], n->point[1], theState (v));
+	       n->point[0], n->point[1], theState (v),
+	       VERTEX_FORWARD_EDGE (v)->is_round ? "Round" : "Line",
+	       VERTEX_FORWARD_EDGE (v)->is_round ? v->radius : 0);
     }
   while ((v = NEXT_VERTEX(v)) != s);
 }
@@ -2829,21 +2831,6 @@ flip_cb (const BoxType * b, void *cl)
   return 1;
 }
 
-#ifndef DEBUG
-static void
-pline_dump (VNODE * v)
-{
-  VNODE *s;
-
-  s = v;
-  do
-    {
-      pcb_fprintf (stderr, "%mn %mn  - %s, radius %mn\n", v->point[0], v->point[1], v->is_round ? "Round" : "Line", v->is_round ? v->radius : 0);
-    }
-  while ((v = v->next) != s);
-}
-#endif
-
 static void
 test_polyInvContour (void)
 {
